src/ice.js

```diff
@@ -802,10 +863,10 @@
 			},
 
 			getUserStyle: function (userid) {
-				var styleIndex = null;
-				if (this._userStyles[userid]) styleIndex = this._userStyles[userid];
-				else styleIndex = this.setUserStyle(userid, this.getNewStyleId(userid));
-				return styleIndex;
+				var style = null;
+				if (this._userStyles[userid]) style = this._userStyles[userid];
```

Comment:
affreux, utilise les {}

Reply:
ou une ternaire, au choix.

Reply:
style = this._userStyles[userid] || null

Reply:
ternaire